code/datums/martial/boxing.dm

```diff
@@ -281,6 +296,66 @@
 	honorable_boxer = FALSE
 	boxing_traits = list(TRAIT_BOXING_READY, TRAIT_STRENGTH, TRAIT_STIMMED)
 
+/// Hunter Boxing: for the uncaring, completely deranged one-spacer ecological disaster.
+/// The honor check accepts boxing ready targets, OR various biotypes as valid targets. Uses a special crit effect rather than the standard one (against monsters).
+/// I guess technically, this allows for lethal boxing. If you want.
+/datum/martial_art/boxing/hunter
+	name = "Hunter Boxing"
+	id = MARTIALART_HUNTER_BOXING
+	pacifist_style = FALSE
+	default_damage_type = BRUTE
+	boxing_traits = list(TRAIT_BOXING_READY)
+	/// The mobs we are looking for to pass the honor check
+	var/honorable_mob_biotypes = MOB_BEAST | MOB_SPECIAL | MOB_PLANT | MOB_BUG
+
+/datum/martial_art/boxing/hunter/honor_check(mob/living/possible_boxer)
+	// Unlike normal, this isn't an early return, because we're looking for MOB_HUMANOID as an early return. We DO want to return true for any instance where our target is boxing ready.
+	if(HAS_TRAIT(possible_boxer, TRAIT_BOXING_READY))
+		return TRUE
+
+
+	if(possible_boxer.mob_biotypes & MOB_HUMANOID && !istype(possible_boxer, /mob/living/simple_animal/hostile/megafauna)) //We're after animals, not people. Unless they want to box. (Or a megafauna)
+		return FALSE
+
+	if(possible_boxer.mob_biotypes & honorable_mob_biotypes) //We're after animals, not people
+		return TRUE
+
+	return FALSE //rather than default assume TRUE, we default assume FALSE. After all, there could be mobs that are none of our biotypes and also not humanoid. By default, they would be valid for being boxed if TRUE.
+
+// Our hunter boxer applies a rebuke and double damage against the target of their crit. If the target is humanoid, we just perform our regular crit effect instead.
+
+/datum/martial_art/boxing/hunter/crit_effect(mob/living/attacker, mob/living/defender, armor_block = 0, damage_type = STAMINA, damage = 0)
+	if(defender.mob_biotypes & MOB_HUMANOID && !istype(defender, /mob/living/simple_animal/hostile/megafauna))
+		return ..() //Applies the regular crit effect if it is a normal human, and not a megafauna
+
+	var/first_word_strike = pick("Extinction", "Brutalization", "Explosion", "Adventure", "Thunder", "Lightning", "Sonic", "Atomizing", "Whirlwind", "Tornado", "Shark", "Falcon")
```

Comment:
Maybe should be either a property of the datum or a global list.
As is, this list will be re-instantiated every time you crit someone which is a bit wasteful.
If it's either global or on the datum, admins can edit it for fun (probably easier if it is on the datum).